specificatie/domain.yaml

```diff
@@ -338,6 +471,16 @@ components:
           type: "string"
           description: |
                         Het domein waartoe de identificatie behoort.
+        indicatieVervallen:
+          type: "boolean"
+          description: |
+                        Vervallen objecten zijn opvraagbaar met de stand van zaken direct voordat het object vervallen is verklaard.
+                        Deze informatie is nodig om de filiatie-"boom" af te kunnen lopen en inzicht te krijgen in betrokken appartementsrechten en grondpercelen.
```

Comment:
@CathyDingemanse ik vind de tweede zin van de description wat merkwaardig. Veronderstelt bij de gebruiker kennis van het begrip "filiatie" en dat deze begrijpt wat het aflopen van de filiatie-"boom" is.
Bovendien is indicatieVervallen niet specifiek nodig om deze filiatie-boom af te lopen, dat kan op basis van isOntstaanUit. Komt mij eigenlijk meer over als documentatie voor de berichtontwerper (waarom heb ik dit toegevoegd) dan voor de API gebruiker.

Reply:
Mijn vermoeden is dat een gebruiker die te maken heeft met vervallen objecten weet wat filiatie (met bijbehorende boom)  is. Dat is namelijk ook de enige reden dat er überhaupt vervallen objecten opgevraagd kunnen worden. 

Los daarvan was ik zelf ook niet happy met deze description en heb ik Cathy in https://github.com/VNG-Realisatie/Haal-Centraal-BRK-bevragen/issues/42#issuecomment-734422007 gevraagd om hierover mee te denken. Ik was vergeten haar te taggen, dus die vraag heeft haar waarschijnlijk niet bereikt. Ik heb nu een comment met tag naar Cathy en Hilde toegevoegd.